api4/user_test.go

```diff
@@ -2562,6 +2562,8 @@ func TestCreateUserAccessToken(t *testing.T) {
 		th := Setup().InitBasic()
 		defer th.TearDown()
 
+		th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableUserAccessTokens = true })
```

Comment:
0/5, but for things like this, I've been adding a test-specific setup function like [this](https://github.com/mattermost/mattermost-server/blob/master/app/post_metadata_test.go#L57) to prevent having to repeat configuration logic across subtests

Reply:
Definitely agree in general! In this case I'm subtly tuning the setup each time to test various code paths, though `*cfg.ServiceSettings.EnableUserAccessTokens = true` perhaps being the least varied one.